Mathlib/FieldTheory/Galois/Basic.lean

```diff
@@ -539,7 +539,7 @@ instance (F K : Type*) [Field F] [Field K] [h : IsQuadraticExtension F K] :
 /--
 A quadratic extension has abelian Galois group.
 -/
-instance (F K : Type*) [Field F] [Field K] [IsQuadraticExtension F K] :
+instance (F K : Type*) [Field F] [Field K] [Algebra F K] [h : IsQuadraticAlgebra F K] :
```

Comment:
You don't need to name the instance here, right?

Reply:
I decided to name the instances that might be useful, so that include the three instances in this file. I don't have any applications in view but I thought it was a safe choice.